tarantula.lic

```diff
@@ -0,0 +1,140 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#tarantula
+=end
+
+custom_require.call(%w(common))
+
+class Tarantula
+  include DRC
+  
+  def initialize
+    settings = get_settings
+    @tarantula = settings.tarantula
+    @tarantula_noun = settings.tarantula_noun.nil? ? 'biomechanical tarantula' : settings.tarantula_noun
+    @exclude = settings.tarantula_excluded_skills
+    UserVars.tarantula_last_use ||= Time.now - 600
+    @last = UserVars.tarantula_last_skillset
+    @last ||= check_last
+    @no_use_scripts = settings.tarantula_no_use_scripts
+    @debug = settings.tarantula_debug.nil? ? false : settings.tarantula_debug
+    @all_skills = { 
+      "Armor" => ['Shield Usage', 'Light Armor', 'Chain Armor', 'Brigandine', 'Plate Armor', 'Defending', 'Conviction'],
+      "Weapon" => ['Parry Ability', 'Small Edged', 'Large Edged', 'Twohanded Edged', 'Small Blunt', 'Large Blunt', 'Twohanded Blunt', 'Slings', 'Bow', 'Crossbow', 'Staves', 'Polearms', 'Light Thrown', 'Heavy Thrown', 'Brawling', 'Offhand Weapon', 'Melee Mastery', 'Missile Mastery', 'Expertise'],
+      "Magic" => ['Lunar Magic', 'Holy Magic', 'Life Magic', 'Elemental Magic', 'Arcane Magic', 'Inner Magic', 'Inner Fire', 'Attunement', 'Arcana', 'Targeted Magic', 'Augmentation', 'Debilitation', 'Utility', 'Warding', 'Sorcery', 'Astrology', 'Summoning', 'Theurgy'],
+      "Survival" =>  ['Evasion', 'Athletics', 'Perception', 'Scouting', 'Stealth', 'Locksmithing', 'Thievery', 'First Aid', 'Outdoorsmanship', 'Skinning', 'Backstab', 'Thanatology'],
+      "Lore" => ['Forging', 'Engineering', 'Outfitting', 'Alchemy', 'Enchanting', 'Scholarship', 'Appraisal', 'Bardic Lore', 'Trading', 'Performance', 'Tactics', 'Empathy'] }
+    
+    passive_loop
+  end
+  
+  def check_last
+    _respond( bold("*** Checking last skillset used ***") ) if @debug
+    bput("study #{@tarantula_noun}", /knowledge of (.*) techniques/, /current set to consume .* experience\./) =~ /knowledge of (.*) techniques/
```

Comment:
💡 Is it 'current set to consume' or 'currently set to consume'?

Reply:
It's current set.  I have bugged the typo in game.